opm/simulators/linalg/bda/Matrix.hpp

```diff
@@ -17,41 +17,83 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef FPGA_MATRIX_HEADER_INCLUDED
-#define FPGA_MATRIX_HEADER_INCLUDED
+#ifndef MATRIX_HEADER_INCLUDED
+#define MATRIX_HEADER_INCLUDED
```

Comment:
This is a rather generic header name and chances are that other use it, too and if that is the case compilation will break as it might already have been defined elsewhere. Please at least add a OPM_ prefix.

Reply:
613cfd2e